### PR TITLE
Find git root

### PR DESCRIPTION
**Pull request by emilrowland**
Added a possibility to find the git root without knowing where the root folder is located if you are in a git project.

### CODE REVIEW

**Comment by emilrowland** [comment]
The automated tests for this project is broken.

File: index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 var fs = require('fs');
 var path = require('path');
+var findUp = require('find-up');
```
Comment (jonschlinkert) [inline index.js]: please use https://github.com/js-cli/node-findup-sync instead. `find-up` is a duplicate library.

Comment (jonschlinkert) [reply index.js]: you should be able to revert changes to `.travis.yml` and node version afterward.

Comment (emilrowland) [reply index.js]: I have updated the package and revert my changes to `.travis.yml` and node version.

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 sudo: false
 os:
   - linux
-  - osx
```
Comment (jonschlinkert) [inline .travis.yml]: Not sure why `osx` was removed, tests do not always pass or fail equally on both.

Comment (jonschlinkert) [reply .travis.yml]: nvm I forgot this was reverted.

**Pull request merged**